### PR TITLE
Add dynamics to website

### PR DESCRIPTION
**Pull request by abelagza**
# Portfolio: mobile menu 

#### In this milestone we:

- Implemented JS for the burger button.
- Fixed HTML tags
- Fixed CSS styling sheet
- Fixed linters

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -41,6 +41,34 @@
...
+    right: 0%;
+    transition: all 0.5s ease-in-out;
+  }
+
+  header .nav-bar .buttons-c .h-bttn {
+    text-decoration: none;
+    color: white;
+    margin: 16px 0;
+  }
+
```
Comment (omar-muhamad) [inline style.css]: - [ ] Please make the mobile menu background cover the whole page to be like the Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image | image  |

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi Team,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by abelagza** [comment]
# Portfolio: mobile menu
#### In this milestone I:

- Fixed website styling according to Figma's.

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**